Ingestion/templates/datahub-executor.ecs.template.yaml

```diff
@@ -0,0 +1,668 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::LanguageExtensions
+Metadata:
+    AWS::CloudFormation::Interface:
+      ParameterGroups:
+        - Label:
+            default: Network
+          Parameters:
+            - VPCID
+            - SubnetID
+        - Label:
+            default: AWS Environment
+          Parameters:
+            - AwsRegion
+        - Label:
+            default: DataHub Info
```

Comment:
Maybe we call this "Configurations"

Reply:
Renamed these groups as follows:

- Network Parameters
- DataHub Parameters
- Worker Task Parameters